sdk-playground/packages/client/src/actions/authActions.ts

```diff
@@ -43,7 +43,7 @@ export const start = async () => {
 			'rpc.voice.write',
 			'rpc.voice.read',
 			// "webhook.incoming",
-			discordSdk.guildId == null ? 'dm_channels.read' : null, // This scope requires approval from Discord.
+			// discordSdk.guildId == null ? 'dm_channels.read' : null, // This scope requires approval from Discord.
```

Comment:
was this meant to be included?

Reply:
yes because it just breaks the activity if you don't have the scope, and you need to do special stuff to get the scope